OptimizelySDK/Odp/OdpSegmentApiManager.cs

```diff
@@ -132,62 +131,85 @@ public class OdpSegmentApiManager : IOdpSegmentApiManager
                 return null;
             }
 
-            return parsedSegments.Data.Customer.Audiences.Edges.
-                Where(e => e.Node.State == BaseCondition.QUALIFIED).
-                Select(e => e.Node.Name).ToArray();
+            return parsedSegments.Data.Customer.Audiences.Edges
+                .Where(e => e.Node.State == BaseCondition.QUALIFIED)
+                .Select(e => e.Node.Name)
+                .ToArray();
         }
 
         /// <summary>
         /// Build GraphQL query for getting segments 
         /// </summary>
-        /// <param name="userKey">'vuid' or 'fs_user_id key'</param>
+        /// <param name="userKey">Either `vuid` or `fs_user_id` key</param>
         /// <param name="userValue">Associated value to query for the user key</param>
         /// <param name="segmentsToCheck">Audience segments to check for experiment inclusion</param>
         /// <returns>GraphQL string payload</returns>
         private static string BuildGetSegmentsGraphQLQuery(string userKey, string userValue,
             IEnumerable segmentsToCheck
         )
         {
-            var query =
-                "query($userId: String, $audiences: [String]) {customer(|userKey|: $userId){audiences(subset: $audiences) {edges {node {name state}}}}}".
-                    Replace("|userKey|", userKey);
-
-            var variables =
-                "\"variables\": { \"userId\": \"|userValue|\", \"audiences\": |audiences| }".
-                    Replace("|userValue|", userValue).
-                    Replace("|audiences|", JsonConvert.SerializeObject(segmentsToCheck));
-
-            return $"{{ \"query\": \"{query}\", {variables} }}";
+            return
+                @"{
+                    ""query"": ""{
+                        query($userId: String, $audiences: [String]) {
+                            {
+                                customer({userKey}: $userId) {
+                                    audiences(subset: $audiences) {
+                                        edges {
+                                            node {
+                                                name
+                                                state
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }"", 
+                    ""variables"" : {
+                        ""userId"": ""{userValue}"",
+                        ""audiences"": {audiences}
+                    }
+                }"
+                    .Replace("{userKey}", userKey)
+                    .Replace("{userValue}", userValue)
+                    .Replace("{audiences}", JsonConvert.SerializeObject(segmentsToCheck));
```

Comment:
please use strings.Join instead of serialization.

Reply:
I think the Join() to create a `["string_1", "string_2"]` will not read as well.

What do you think?